Bottom: 0f6b0b90d7739c61acae47649b0c406540570e73
Top:    a0d518f6267bdf08ee99afc306ffb7a4fd257fd2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-17 01:01:10 +0100

Refresh of fix-crash-in-smashbufferelemen

---

diff --git a/src/buffer.c b/src/buffer.c
index 8793f50..c048a59 100644
--- a/src/buffer.c
+++ b/src/buffer.c
@@ -926,7 +926,7 @@ SmashBufferElement (BufferTypePtr Buffer)
   }
   END_LOOP;
   FreeElementMemory (element);
-  free (element);
+  FreeElement (element);
   return (true);
 }
